tests/e2e/test_tools.py

```
@@ -151,6 +152,36 @@ async def test_tool_search(mcp_server, patient_id):
         raise
 
 
+@pytest.mark.asyncio
+async def test_tool_search_toon_format(mcp_server, patient_id):
+    request_payload = {"type": "Patient", "searchParam": {"_id": patient_id}}
+    logger.debug("[TEST REQUEST] search toon:", request_payload)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)